src/query-builder/function-module.ts

```diff
@@ -198,11 +285,112 @@ export class FunctionModule<DB, TB extends keyof DB> {
     })
   }
 
+  /**
+   * Calls the `count` function with `*` or `table.*` as argument.
+   *
+   * When called with `*` as argument, this sql function counts the number of rows,
+   * nulls included.
+   *
+   * For counting rows with non-null values in a given column (`count(column)`),
+   * see {@link count}.
+   *
+   * For additional functionality such as filtering and window functions, refer
+   * to {@link AggregateFunctionBuilder}. An instance of this builder is returned
+   * when calling this function.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * const { count } = db.fn
```

Comment:
I think you meant `countAll` here otherwise it might not work 😅

Reply:
@zmilonas Nice catch! Wanna submit a PR fixing this?

Reply:
Sure thing! Here you go: https://github.com/koskimas/kysely/pull/303